OracleEssbase/dockerfiles/21.1.0/Dockerfile

```diff
@@ -0,0 +1,125 @@
+    $JAVA_HOME/bin/java -jar /u01/$ESSBASE_JAR -silent -responseFile /u01/install.file \
+                                               -invPtrLoc /u01/oraInst.loc \
+                                               -jreLoc $JAVA_HOME \
+                                               -ignoreSysPrereqs -force -novalidation \
+                                               ORACLE_HOME=$ORACLE_HOME INSTALL_TYPE="Essbase Distribution" && \
+    rm /u01/${ESSBASE_JAR} /u01/install.file && \
+    rm -rf ${ORACLE_HOME}/cfgtoollogs && \
+    rm -rf ${ORACLE_HOME}/.inventory/logs
+
+# Apply Patches
+COPY --chown=oracle:oracle patches /u01/patches
+RUN /u01/patches/apply-patches.sh && \
+    rm -rf /u01/patches
+
+#
+# Final image stage
+#
+FROM base
+
+ARG ESSBASE_VERSION=21.1.0
+RUN mkdir /etc/essbase && \
+    chmod 755 /etc/essbase && \
+    echo "Oracle Essbase Container ${ESSBASE_VERSION}" > /etc/essbase/essbase-version-info && \
+    chmod 644 /etc/essbase/essbase-version-info
+
+
+# Environment variables required for this build (do NOT change)
+# (DOMAIN_NAME/HOME are reset here, to avoid FMW Infra environment leaking through)
+# -------------------------------------------------------------
+ENV DOMAIN_NAME="${DOMAIN_NAME:-essbase_domain}" \
+    DOMAIN_ROOT="${DOMAIN_ROOT:-/u01/config/domains}" \
+    DOMAIN_HOME="${DOMAIN_ROOT:-/u01/config/domains}/${DOMAIN_NAME:-essbase_domain}" \
+    ARBORPATH="${ARBORPATH:-/u01/data/essbase}" \
+    TMP_DIR="${TMP_DIR:-/u01/tmp}" \
+    ADMIN_USERNAME="${ADMIN_USERNAME:-admin}" \
+    ADMIN_PASSWORD="${ADMIN_PASSWORD:-welcome1}" \
+    DATABASE_TYPE="${DATABASE_TYPE:-ORACLE}" \
+    DATABASE_CONNECT_STRING="${DATABASE_CONNECT_STRING:-rcu-db:1521/PDBORCL}" \
+    DATABASE_ADMIN_USERNAME="${DATABASE_ADMIN_USERNAME:-sys}" \
+    DATABASE_WAIT_TIMEOUT="${DATABASE_WAIT_TIMEOUT:-0}" \
+    DATABASE_SCHEMA_PREFIX="${DATABASE_SCHEMA_PREFIX:-ESS1}" \
+    CREATE_SCHEMA="${CREATE_SCHEMA:-TRUE}" \
+    DROP_SCHEMA="${DROP_SCHEMA:-FALSE}" \
+    MACHINE_NAME_PREFIX="${MACHINE_NAME_PREFIX:-essbase_machine}" \
+    NODE_MANAGER_PORT="${NODE_MANAGER_PORT:-9556}" \
+    ADMIN_SERVER_PORT="${ADMIN_SERVER_PORT:-7001}" \
+    ADMIN_SERVER_SSL_PORT="${ADMIN_SERVER_SSL_PORT:-7002}" \
+    MANAGED_SERVER_PORT="${MANAGED_SERVER_PORT:-9000}" \
+    MANAGED_SERVER_SSL_PORT="${MANAGED_SERVER_SSL_PORT:-9001}" \
+    ESSBASE_CLUSTER_SIZE="${ESSBASE_CLUSTER_SIZE:-1}" \
+    AGENT_PORT="${AGENT_PORT:-1423}" \
+    AGENT_SSL_PORT="${AGENT_SSL_PORT:-6423}" \
+    ESSBASE_SERVER_MIN_PORT="${ESSBASE_SERVER_MIN_PORT:-30768}" \
+    ESSBASE_SERVER_MAX_PORT="${ESSBASE_SERVER_MAX_PORT:-31768}" \
+    ENABLE_EAS="${ENABLE_EAS:-FALSE}" \
+    EAS_SERVER_PORT="${EAS_SERVER_PORT:-9100}" \
+    EAS_SERVER_SSL_PORT="${EAS_SERVER_SSL_PORT:-9101}" \
+    IDENTITY_PROVIDER="${IDENTITY_PROVIDER:-EMBEDDED}"
+
+USER oracle
+COPY --from=builder --chown=oracle:oracle /u01 /u01
+COPY --chown=oracle:oracle container-scripts /u01/container-scripts
```

Comment:
It's good practice to `chmod +x` your script within the `Dockerfile` just in case this repo was is cloned to a OS that doesn't support POSIX flags and the file loses its executable status.

Alternatively, change your `CMD` to run `bash -c <script>`

Reply:
Fair enough...added task to chmod +x the target scripts in the container after copy.